dev/com.ibm.ws.cdi.api_fat/test-applications/cdiCurrentTest.war/src/com/ibm/ws/cdi12/test/common/web/CDICurrentTestServlet.java

```diff
@@ -37,4 +46,65 @@ public void testCDICurrent() {
 
     }
 
+    //This test is not used as calling CDI.current() from a new thread without a managed executor service is not supported.
+    /*
+    public void testCDICurrentInNewThread() throws Exception {
+
+        assertNotNull(CDI.current()); //Test outside a new thread just for completeness.
+
+        new Thread(() -> {
+            CDI cdi = CDI.current();
+            System.out.println("GREP + " + CDI.current());
+            if (cdi != null) {
+                CDICurrentTestServlet.wasCDICurrentFound = true;
+            } else {
+                CDICurrentTestServlet.wasCDICurrentFound = false;
+            }
+        }).start();
+
+        int i = 10;
+
+        while (i > 0) {
+            i--;
+            if (wasCDICurrentFound != null) {
+                assertTrue("CDI.current returned null when called in a new Thread", wasCDICurrentFound);
+                return;
+            }
+            Thread.sleep(5);
+        }
+    }
+    */
```

Comment:
No need to keep this whole test but a good explaination in a comment would still be useful.